R/date_helpers.R

```diff
@@ -24,7 +24,7 @@ as_time_or_date <- function(x) {
 #' @param x Date or POSIXct
 #' @noRd
 regularize_date <- function(x) {
-  stopifnot(class(x)[1] %in% c("POSIXct", "Date"))
+  stopifnot(inherits(x,c("POSIXct", "Date")))
```

Comment:
Nice! I wasn't aware that `inherits()` can have multiple classes. 

Can you follow the style guide and add a space after `x`?